modules/tensor_mechanics/doc/content/documentation/systems/Modules/TensorMechanics/GeneralizedPlaneStrain/index.md

```diff
@@ -1,6 +1,45 @@
 # GeneralizedPlaneStrain Action
 
 Sets up a generalized plane strain model. This action can be generated using the
-[Modules/TensorMechanics/Master](../Master) action.
+[TensorMechanics Master action](../Master).
 
-!syntax parameters /Modules/TensorMechanics/Master/CommonTensorMechanicsAction
```

Comment:
Include  from your theory write up the information on the scalar strain and the implementation here as well (!include.....)